wesnoth-l.po

```diff
@@ -2967,6 +2972,12 @@ msgid ""
 "regroup. It be looking like we really did disrupt Weldyn’s chain of command, "
 "least for now."
 msgstr ""
+"Nevím, co se stalo s Helikromem, neviděl jsem ani to, jak dopad Maddokův "
+"syn. Já, Harper a pár nás zbylejch jsme vzali nohy na ramena. Teď míříme "
+"zpátky na sever, k Carcynu a Elensefaru. Zdá se, že vojenský hlídky jsou ve "
```

Comment:
```suggestion
"zpátky na sever, ke Carcynu a Elensefaru. Zdá se, že vojenský hlídky jsou ve "
```

Reply:
Nějak mi to ke k tomu nesedí. Má to být jako ke městu?

Reply:
dvě souhlásky **k** mi bez **e** připadají jako koktání

Reply:
Aha. Já jsem dosud předpokládal výslovnost jména města Carcyn s C. Jinak bych to býval změnil už v SoA. Klidně můžeme jít touto cestou, ale pak bych to fakt změnil i graficky na Karcyn.

Reply:
Aha, s **C** jsem nepočítal. A protože nevím, jak se to má vyslovovat, a na webu jsem našel jen ruský Caricyn, který je samozřejmě s **C**, tak bych tam nechal to vaše **k**.
Promiňte to zdržení.